policyengine_us/variables/gov/states/ny/tax/income/adjusted_gross_income/ny_agi_subtractions.py

```diff
@@ -15,24 +15,10 @@ class ny_agi_subtractions(Variable):
     defined_for = StateCode.NY
 
     def formula(tax_unit, period, parameters):
```

Comment:
```suggestion
    adds = "gov.states.ny.tax.income.agi.subtractions.sources"
```

Reply:
In this case we dont need a formula - the adds will sum up the list values to the entity of the variable